migrator/migrations/n_04_to_n_05_postgres_images/postgres/store.go

```diff
@@ -1,5 +1,5 @@
 // This file was originally generated with
-// //go:generate cp ../../../../central/image/datastore/store/postgres/store.go store_impl.go
+// //go:generate cp ../../../../central/image/datastore/store/postgres/store.go store.go
```

Comment:
Seems like most of these files are called postgres_plugin.  Why is this one different?

Reply:
postgres_plugin are auto generated and then unlinked. The store.go means the entire migration is customized.